lsp-bridge.el

```diff
@@ -547,8 +547,7 @@ Then LSP-Bridge will start by gdb, please send new issue with `*lsp-bridge*' buf
                (if (minibufferp) (window-buffer (minibuffer-selected-window))
                  (current-buffer))
 
-             (let ((snippet-fn (and (string= kind "Snippet")
```

Comment:
Why delete (string= kind "Snippet") ?

Reply:
But if you delete this line, snippet-fn always is true, we can't do lsp-bridge--apply-text-edits if Kind is auto-import.

Reply:
I see. I'll push a commit to fix it.

Reply:
@manateelazycat done